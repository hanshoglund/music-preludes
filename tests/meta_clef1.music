let
    part1 = clef f $ staccato $ scat [c_,g_,c,g_]
    part2 = clef c $ staccato $ scat [ab_,eb,d,a]
    part3 = clef g $ staccato $ accentLast $ scat [g,fs,e,d]
in compress 8 $ part1 |> part2 |> part3


