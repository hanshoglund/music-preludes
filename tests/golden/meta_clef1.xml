<?xml version='1.0' ?>
<score-partwise>
  <movement-title></movement-title>
  <identification>
    <creator type="composer"></creator>
  </identification>
  <part-list>
    <score-part id="P1">
      <part-name></part-name>
    </score-part>
  </part-list>
  <part id="P1">
    <measure number="1">
      <attributes>
        <key>
          <fifths>0</fifths>
          <mode>major</mode>
        </key>
      </attributes>
      <attributes>
        <divisions>768</divisions>
      </attributes>
      <direction>
        <direction-type>
          <metronome>
            <beat-unit>quarter</beat-unit>
            <per-minute>60</per-minute>
          </metronome>
        </direction-type>
      </direction>
      <attributes>
        <time>
          <beats>4</beats>
          <beat-type>4</beat-type>
        </time>
      </attributes>
      <attributes>
        <clef>
          <sign>F</sign>
          <line>4</line>
        </clef>
      </attributes>
      <note>
        <pitch>
          <step>C</step>
          <alter>0.0</alter>
          <octave>3</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>G</step>
          <alter>0.0</alter>
          <octave>3</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>C</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>G</step>
          <alter>0.0</alter>
          <octave>3</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <attributes>
        <clef>
          <sign>C</sign>
          <line>3</line>
        </clef>
      </attributes>
      <note>
        <pitch>
          <step>A</step>
          <alter>-1.0</alter>
          <octave>3</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>E</step>
          <alter>-1.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>D</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>A</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
    </measure>
    <measure number="2">
      <attributes>
        <time>
          <beats>2</beats>
          <beat-type>4</beat-type>
        </time>
      </attributes>
      <attributes>
        <clef>
          <sign>G</sign>
          <line>2</line>
        </clef>
      </attributes>
      <note>
        <pitch>
          <step>G</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>F</step>
          <alter>1.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>E</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
          </articulations>
        </notations>
      </note>
      <note>
        <pitch>
          <step>D</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations>
            <staccato />
            <accent />
          </articulations>
          <technical />
        </notations>
      </note>
    </measure>
  </part>
</score-partwise>
